pyslim/methods.py

```diff
@@ -608,6 +608,20 @@ def annotate_tables(tables, model_type, tick, cycle=None, stage="early", referen
     if reference_sequence is not None:
         tables.reference_sequence.data = reference_sequence
 
+def max_slim_mutation_id(ts):
+    '''
+    Returns the largest SLiM mutation ID in this tree sequence. If you want to
+    add more mutations to your tree sequence using :func:`msprime.sim_mutations`,
+    you may need to specify the parameter `next_id` in your 
+    :class:`msprime.SLiMMutationModel`. Setting `next_id` correctly is necessary 
+    if you want your tree sequence to be readable by SLiM.
```

Comment:
```suggestion
    Returns the largest SLiM mutation ID in this tree sequence. This is useful because
    if you want to add more mutations to your tree sequence using :func:`msprime.sim_mutations`,
    you may need to specify the parameter `next_id` in your 
    :class:`msprime.SLiMMutationModel` to be larger than any existing mutation IDs.
    Setting `next_id` equal to the output of this function
    will allow the mutated tree sequence to be read in by SLiM. 
```